### PR TITLE
Adds archive information to chandrayaan2 template

### PR DESCRIPTION
**Pull request by amystamile-usgs**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Infrastructure change (changes to things like CI or the build system that do not impact users)

## Checklist:

- [ ] I have read and agree to abide by the [Code of Conduct](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/Code-Of-Conduct.md)
- [ ] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation and I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] I have added myself to the [.zenodo.json](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/.zenodo.json) document.
- [ ] I have added my user impacting change to the [CHANGELOG.md](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CHANGELOG.md) document. 

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [ ] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

File: isis/appdata/import/PDS4/Chandrayaan2TMC2.tpl [hunk 1]
```diff
@@ -67,15 +67,42 @@ Object = IsisCube
   End_Object
 
   Group = Instrument
-    SpacecraftName            = {{ Product_Observational.Observation_Area.Investigation_Area.name }}
+    SpacecraftName            = {{ capitalize(Product_Observational.Observation_Area.Investigation_Area.name) }}
```
Comment (Kelvinrr) [inline isis/appdata/import/PDS4/Chandrayaan2TMC2.tpl:70]: Why is this needed?

**Review by Kelvinrr — APPROVED** [review]
just a question in line, I think it looks good overall

**Review by Kelvinrr — DISMISSED** [review]

File: isis/appdata/import/PDS4/Chandrayaan2TMC2.tpl (hunk 1 above)
Comment (amystamile-usgs) [reply isis/appdata/import/PDS4/Chandrayaan2TMC2.tpl:70]: Because the 2019 and 2024 labels differ in spacecraft name: Chandrayaan-2 vs chandrayaan-2. Trying to keep them consistent.

**Review by Kelvinrr — APPROVED** [review]
